openshift/4.0/scripts/oc-sonar-scan.sh

```diff
@@ -52,6 +52,7 @@ CMD_TEST="npm run coverage"
 # Begin analysis
 #
 CMD_SONAR_SCAN="sonar-scanner \
+  -D'sonar.projectName=pims-frontend[${TARGET}]' \
```

Comment:
the project name was always defaulting to pims-frontend[dev] this should fix that.